spec/spec_helper.rb

```diff
@@ -1,3 +1,5 @@
 require 'codeclimate-test-reporter'
 
+require_relative '../lib/gobstones_server'
```

Comment:
because a conflict between ActiveSupport::JSON and the JSON gem. For more info ask @flbulgarelli :p